dimagi/pages/urls/redirect.py

```diff
@@ -24,6 +24,7 @@
     url(r'^confidant.shtml$', redirect.page('commcare')),
     url(r'^commtrack/$', redirect.page('commcare')),
     url(r'^collaborate/openrosa/$', redirect.page('commcare')),
+    url(r'^referral-commcare/$', redirect.page('commcare')),
```

Comment:
this uses the usual redirect which is not a `307`. `307` is a temporary redirect, which is what we want